src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -259,6 +337,31 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         /// ]]></format>
         /// </example>
         public static ValueMappingEstimator MapValue(
+            this TransformsCatalog.ConversionTransforms catalog,
+            IDataView lookupMap, string keyColumnName, string valueColumnName, string outputColumnName, string inputColumnName = null)
```

Comment:
>outputColumnName, string inputColumnName = null) [](start = 86, length = 48)

In other places, we tend to have outputColumnName be first argument and inputColumnName for the second. Here is an example.
```charp
        public static ValueToKeyMappingEstimator MapValueToKey(this TransformsCatalog.ConversionTransforms catalog,
            string outputColumnName,
            string inputColumnName = null,
```

Reply:
I think we have teh output column name, then other required arguments.

---
In reply to: [266129712](https://github.com/dotnet/machinelearning/pull/2959#discussion_r266129712) [](ancestors = 266129712)

Reply:
Thank you for pointing that out, I will make sure we respect that in other places too. 

---
In reply to: [266549273](https://github.com/dotnet/machinelearning/pull/2959#discussion_r266549273) [](ancestors = 266549273,266129712)